s4/str_conditional.rb

```diff
@@ -0,0 +1,21 @@
+# include?()
+text = "Snow white"
+print ".include?(\"S\") = "; p text.include?("S")
+print ".include?(\"now\") = "; p text.include?("now")
+print ".include?(\" \") = "; p text.include?(" ")
+print ".include?(\"black\") = "; p text.include?("black")
+print ".include?(\"snow\") = "; p text.include?("snow")
+print ".downcase.include?(\"snow\") = "; p text.downcase.include?("snow")
+puts
+
+# .emty? && .nil?
+v = ""
+print "v = "
+p v
+print "v.empty? = "; p v.empty?
```

Comment:
What's the difference between `empty?` and `nil?` ?

Reply:
That `empty?` will only return true if it is empty, while `nil?` will only return true if it is _null_.